azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -732,6 +733,19 @@ public void shutdownNow() {
     logger.warn("Shutting down FlowRunnerManager now...");
     this.executorService.shutdownNow();
     this.triggerManager.shutdown();
+    deleteExecutionDirectory();
```

Comment:
My mistake, I mis-interpreted the call to the shotdown method for the triggerManager as the call for the FlowRunnerManager.